Volha_Ivanova/Module_4_Framework/Layers_Runner_BusinessObjects/my-app/src/test/java/com/epam/cdp/volha_ivanova/task1/test/TwoUsersSignInTest.java

```diff
@@ -0,0 +1,81 @@
+package com.epam.cdp.volha_ivanova.task1.test;
+
+import org.openqa.selenium.Alert;
+import org.openqa.selenium.By;
+import org.openqa.selenium.NoAlertPresentException;
+import org.openqa.selenium.UnhandledAlertException;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.Wait;
+import org.openqa.selenium.support.ui.WebDriverWait;
+import org.testng.annotations.Parameters;
+import org.testng.annotations.Test;
+import com.epam.cdp.volha_ivanova.task1.pages.EmailListPage;
+import com.epam.cdp.volha_ivanova.task1.pages.SignInPage;
+import com.epam.cdp.volha_ivanova.task1.pages.StartPage;
+
+public class TwoUsersSignInTest extends BaseClassTest{
+	/**
+	 * @author Volha_Ivanova Browser Chrome 57.0.2987.98 MacOS.
+	 */	
+	@Parameters({"First_user_email", "First_user_password", "Second_user_email", "Second_user_password" })
+	@Test
+	public void testOneCanLoginGithub(String First_user_email, String First_user_password, String Second_user_email, String Second_user_password) {
+		/**
```

Comment:
What reason of using comments in test? I suppose your code should be readable enough to avoid using of useless comments.
Currently I understand steps of test without the comments